RELEASE_NOTES.md

```diff
@@ -2,6 +2,16 @@
 
 This section contains past updates we've sent to customers via Discord.
 
+## 0.17.4 (November 10, 2023)
+
+- 💿 include mixed-mode capability (not yet being used)
+
```

Comment:
```suggestion
We rewrote how Dart code is executed on iOS in this release.  We haven't yet enabled the new faster "mixed-mode" execution so there should be no observable change, but the faster VM is now included and in partial use.  Because this was such a large (multi-month!) change, there could be unanticipated changes on iOS devices.  If you see anything surprising, please let us know!
```